tests/common/gcp_type/billing_account_test.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2018 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests the Billing Account resource"""
+
+import mock
+import unittest
+
+from tests.unittest_utils import ForsetiTestCase
+from google.cloud.forseti.common.gcp_type import billing_account
+from google.cloud.forseti.common.gcp_type import organization
+from google.cloud.forseti.common.gcp_type import resource_util
+
+
+class BillingAccountTest(ForsetiTestCase):
+    """Test Billing Account resource."""
+
+    def setUp(self):
+        self.org_234 = organization.Organization(
+            '234',
+            display_name='Organization 234',
+            full_name='organization/234/',
+            data='fake_org_data_234')
+
+    def test_getters_are_correct(self):
+        """Tests whether the Billing Account methods return expected data."""
+        billing_acct = billing_account.BillingAccount(
+            '000000-111111-222222',
+            display_name='My Billing Account',
+            parent=self.org_234)
+
+        self.assertEqual('000000-111111-222222', billing_acct.id)
+        self.assertEqual('billing_account', billing_acct.type)
+        self.assertEqual('billingAccounts/000000-111111-222222',
+                         billing_acct.name)
+        self.assertEqual('My Billing Account', billing_acct.display_name)
+        self.assertEqual(self.org_234, billing_acct.parent)
+
+    def test_create_from_resource_utils(self):
```

Comment:
This test doesn't seem to be testing anything from billing_account.py. Should it be in resources instead?